.ci/static-checks.sh

```diff
@@ -327,7 +347,8 @@ static_check_versions()
 	if [ -n "$have_yamllint_cmd" ]; then
 		eval "$yamllint_cmd" "$db"
 	else
-		info "Cannot check versions as $yamllint_cmd not available"
+		info "Installing yamllint"
```

Comment:
Should we do this check first, and then install, and then re-try the `eval` iyswim - otherwise, I think on first invocation it will install yamllint, but skip the requested check??